automation_tools/scripts/ga-migration/README.md

```diff
@@ -8,12 +8,34 @@ $ source ~/.virtualenvs/ga-migration/bin/activate
 $ pip install -r requirements.txt
 ```
 
-## Run
+## Migration
 
-```console
-# Migrate specific repo
-$ python main.py --targetpath ../path/to/invenio/repo
+Apply migration patches on a cloned repository:
+
+```bash
+# e.g.
+# git clone https://github.com/inveniosoftware/invenio-charts-js
+python main.py --targetpath=invenio-charts-js
+```
+
+## Full pipeline
 
-# Migrate all repos on config.py
-$ python main.py
```

Comment:
Why have you deleted this option? It's useful to bulk-migrate repos

Reply:
Woops